Notebook to create a dataset

javierhndev opened the pull request:
This notebook generates figures from PySM and save them as 'png'. Those figures can be used to genereate a training dataset. Also a pixell `map` is saved in hdf to keep the `wcs`.

zonca commented:
as we talked:

* use monochrome colormap
* use `execute=False` and a `if not execute exit` to avoid accidental execution
* do a clean rerun of the input data generation from top to bottom 
* save `nbstripout` version in repo and executed version in a gist and link it somewhere
* rerun the training and inference on voyager
* provide 10 test maps to me, either in hdf5 format or in PNG but with a script to convert to pixell enmaps with wcs
* create the spectra
* write a small report either as a md file or a notebook that puts everything together

javierhndev commented:
I've modified the notebooks to generate the monochromatic figures.

In the `202505_generate_galactic_dust_realization.ipynb` I added a last section where I create the image with Pillow and then read it to verify it is OK. I added a link to Gist at the top of the notebook with the executed version.

In the `202505_data_generation.ipynb` I included what I've tried above so now images are wrien using Pillow in a monochromatic scale.

zonca commented:
@javierhndev thanks, can you also add a README to the repo that explain what each notebook does?

even reading your explanation I don't understand why you need 2 notebooks instead of 1.

javierhndev commented:
Sure, I will. Yes, it can be confusing, I'll cange that. The first one is just an exploratory notebook to show what can be done. The second one should be a simple script not a notebook because it barely outputs anything relevant (apart from the dataset).

javierhndev commented:
I kept the notebook as it was (with the new `pillow` analysis) and replaced that second notebook with a script. I also added a README
